package.json

```diff
@@ -3,7 +3,7 @@
   "repository": "https://github.com/segmentio/is-url",
   "version": "1.2.3",
   "scripts": {
-    "test": "make test"
+    "test": "./node_modules/.bin/mocha --reporter spec"
```

Comment:
npm adds everything in `./node_modules.bin` to the PATH at runtime, so there's no need for the fully qualified path. Just `mocha` or `mocha --reporter spec` will suffice.